src/fomac/Slurm.cpp

```
@@ -0,0 +1,140 @@
+  }
+
+  if (std::ranges::find(registeredIds, deviceId) == registeredIds.end()) {
+    throw std::runtime_error("Slurm license '" + deviceId +
+                             "' is not a registered QDMI device ID");
+  }
+  return deviceId;
+}
+
+} // namespace
+
+Device openDeviceFromLicense() {
```

Cppcheck (2.21.0)

[style] The function 'openDeviceFromLicense' is never used.

(unusedFunction)